airflow/providers/cncf/kubernetes/kubernetes_helper_functions.py

```diff
@@ -59,15 +59,13 @@ def create_pod_id(
     dag_id: str | None = None,
     task_id: str | None = None,
     *,
-    max_length: int = 80,
```

Comment:
Can you please extract this as constant while you are at it?

Reply:
done!